core/src/test/scala/integration/kafka/api/ConsumerWithLegacyMessageFormatIntegrationTest.scala

```diff
@@ -113,9 +106,7 @@ class ConsumerWithLegacyMessageFormatIntegrationTest extends AbstractConsumerTes
     prop.setProperty(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG, "false")
```

Comment:
Shall we remove this property and comment? Also, the topic name should be updated.